plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -278,10 +282,15 @@ export class MySQLInstrumentation extends InstrumentationBase<
           values = [_valuesOrCallback];
         }
 
-        span.setAttribute(
-          SemanticAttributes.DB_STATEMENT,
-          getDbStatement(query, format, values)
-        );
+        const dbStatementSerializer =
+          typeof thisPlugin._config.dbStatementSerializer === 'function'
+            ? thisPlugin._config.dbStatementSerializer
+            : thisPlugin._defaultDbStatementSerializer.bind(thisPlugin);
+
+        let dbStatement = getDbStatement(query, format, values);
+        dbStatement = dbStatementSerializer(dbStatement);
```

Comment:
Might be more readable to separate the 2 variables instead of overriding a `let` variable, e.g:
```js
const fullDbStatement = getDbStatement(query, format, values);
const sanitizedDbStatement = dbStatementSerializer(dbStatement);
```